assets/src/pages/types.ts

```diff
@@ -0,0 +1,16 @@
+import { TrackEncoding } from "@membraneframework/membrane-webrtc-js";
+
+// todo Change TrackEncoding to something like EncodingType in "@membraneframework/membrane-webrtc-js"
+const EncodingValues = ["l", "m", "h"] as const;
+type EncodingType = typeof EncodingValues[number];
+
+export const isTrackEncoding = (value: string): value is TrackEncoding =>
+  EncodingValues.includes(value as TrackEncoding);
```

Comment:
Remove when this PR will be merged 

https://github.com/membraneframework/membrane-webrtc-js/pull/14